src/validation/rules/custom/NoDeprecatedCustomRule.ts

```diff
@@ -24,11 +22,9 @@ export function NoDeprecatedCustomRule(context: ValidationContext): ASTVisitor {
       const fieldDef = context.getFieldDef();
       const deprecationReason = fieldDef?.deprecationReason;
       if (fieldDef && deprecationReason != null) {
-        const parentType = context.getParentType();
-        invariant(parentType != null);
         context.reportError(
           new GraphQLError(
-            `The field ${parentType.name}.${fieldDef.name} is deprecated. ${deprecationReason}`,
+            `The field ${fieldDef} is deprecated. ${deprecationReason}`,
```

Comment:
Should this be `Field ${fieldDef} is deprecated.` similar to what you're doing in ValuesOfCorrectTypeRule?